src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -118,14 +118,24 @@ export default function appointmentsReducer(state = initialState, action) {
         pastSelectedIndex: action.selectedIndex,
       };
     case FETCH_PAST_APPOINTMENTS_SUCCEEDED: {
-      const { data, startDate, endDate } = action;
+      const { appointments, requests = [], startDate, endDate } = action;
```

Comment:
Why only the default empty array for requests and not appointments?

Reply:
Appointments should always exist, those are fetched whether we're on the new or old homepage. Requests will only exist when we're on the new homepage, so I'm defaulting it to an empty array to make the concat/filtering in the next step easier.